fe/src/main/java/org/apache/doris/catalog/BrokerTable.java

```diff
@@ -222,7 +238,7 @@ public void readFields(DataInput in) throws IOException {
         }
         columnSeparator = Text.readString(in);
         lineDelimiter = Text.readString(in);
-
+        fileFormat = Text.readString(in);
```

Comment:
If you change here, you should do some check META_VERSION to support old meta imange

Reply:
ok